### PR TITLE
Populate blog from API

### PR DESCRIPTION
**Pull request by Sashikbear**
Hey, guys, please check my work with API and tell me if the code is optimised and readable enough. Thanks!

### CODE REVIEW

**Comment by resdenia** [comment]
Amazing work @Sashikbear   it's advanced level but you did it very nice!!
I would leave several comments and would ask you to go over)
Thank you!

File: blog/scripts/blog.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+            // selecting parts of the new element
+            const blogpostTitle = blogpostElement.querySelector(".blog__title");
+            const blogpostYear = blogpostElement.querySelector(".blog__year");
+            const blogpostType = blogpostElement.querySelector(".blog__type");
+            const blogpostText =
+              blogpostElement.querySelector(".blog__description");
+
+            // giving values from API
+            blogpostTitle.textContent = post.title;
```
Comment (resdenia) [inline blog/scripts/blog.js]: to make it more clear you can check this article 
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment

File: blog/scripts/blog.js [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+            blogpostType.textContent = post.id;
+            blogpostText.textContent = post.body;
+            return blogpostElement;
+          };
+          const attachBlogpost = (post) => {
+            blogList.prepend(addBlogpost(post));
+          };
+          attachBlogpost(post);
+        }
+      });
```
Comment (resdenia) [inline blog/scripts/blog.js:44]: If we are using then syntax it's really important to have ` .catch` for then, to handle error state.  in our case we need to console.log error.

**Review by resdenia — CHANGES_REQUESTED** [review]
Thank you again! everything is good. Just add 2 comments to impprove our project.

File: blog/scripts/blog.js (hunk 2 above)
Comment (Sashikbear) [reply blog/scripts/blog.js:44]: ok, fixing it the following syntax:
 .catch((error) => {
      console.error("Error:", error);
    });

File: blog/scripts/blog.js (hunk 1 above)
Comment (Sashikbear) [reply blog/scripts/blog.js]: If I understood the idea correctly, refactoring with the following syntax:
 const blog = {
          title: post.title,
          year: post.userId,
          type: post.id,
          text: post.body,
        };
        const { title, year, type, text } = blog;

**Review by resdenia — APPROVED** [review]
Amazing work! Can Approve

**Pull request merged**